src/main/java/edu/kis/vh/nursery/containers/IntArrayStack.java

```diff
@@ -0,0 +1,42 @@
+package edu.kis.vh.nursery.containers;
+
+// użycie klawisza ctrl oraz kursora myszki pozwala przejść zarówno do deklaracji
+// jak i implementacji
+
+public class IntArrayStack implements IntContainer {
+    static final int capacity = 12;
+    int total = -1;
+    int[] NUMBERS = new int[capacity];
+
+    public IntArrayStack() {
+    }
+
+    public void countIn(int in) {
+        if (!isFull())
+            NUMBERS[++total] = in;
+    }
+
+    public boolean callCheck() {
+        return total == empty;
```

Comment:
Extracting this constant is done incorrectly. (This is not the same empty as e.g. in peakaboo method)